runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -28,6 +28,7 @@
 import com.google.inject.Provides;
 import java.awt.image.BufferedImage;
 import javax.inject.Inject;
+
```

Comment:
Remove this newline (https://github.com/runelite/runelite/wiki/Code-Conventions check xml for intellij idea here)